### PR TITLE
Actually use standard, instead of eslint

### PR DESCRIPTION
**Pull request by SimenB**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,16 +4,16 @@
...
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "grunt"
   },
   "repository": {
     "type": "git",
     "url": "git://github.com/karma-runner/karma-junit-reporter.git"
   },
   "keywords": [
+    "junit",
```
Comment (SimenB) [inline package.json]: This is `npm` sorting when it changes the file.

File: package.json [hunk 2]
```diff
@@ -24,27 +24,25 @@
...
     "grunt": "^0.4.1",
     "grunt-bump": "^0.3.1",
     "grunt-conventional-changelog": "^1.2.2",
-    "grunt-eslint": "^16.0.0",
     "grunt-npm": "^0.0.2",
+    "grunt-standard": "^1.0.2",
     "load-grunt-tasks": "^3.2.0"
   },
   "contributors": [
+    "Brian Donovan <me@brian-donovan.com>",
```
Comment (SimenB) [inline package.json]: Same as above

File: package.json [hunk 3]
```diff
@@ -4,16 +4,16 @@
   "description": "A Karma plugin. Report results in junit xml format.",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "grunt"
```
Comment (SimenB) [inline package.json]: why not :smile:

File: .travis.yml [hunk 4]
```diff
@@ -1,3 +1,4 @@
+sudo: false
```
Comment (SimenB) [inline .travis.yml]: http://docs.travis-ci.com/user/workers/container-based-infrastructure/

File: .travis.yml [hunk 5]
```diff
@@ -14,6 +15,3 @@ before_install:
 
 before_script:
   - npm install -g grunt-cli
-
```
Comment (SimenB) [inline .travis.yml]: `npm test` does it

File: .travis.yml [hunk 6]
```diff
@@ -14,6 +15,3 @@ before_install:
...
-script:
-  - grunt
```
Comment (SimenB) [inline .travis.yml]: `npm test` does it

**Comment by Dignifiedquire** [comment]
Thanks, but I'd rather keep using eslint directly as I'm using it across the board and it's much easier to integrate orher tooling like different parsers and exceptions for test files etc.

**Comment by SimenB** [comment]
@Dignifiedquire Ok :smile: I changed the PR to just the Travis changes, if those are good on their own

**Comment by Dignifiedquire** [comment]
Looks good, thanks :octocat:

**Pull request merged**